bin/openstack_taster

```diff
@@ -12,12 +12,12 @@ settings = {}
 ARGV << '-h' if ARGV.empty?
 
 parser = OptionParser.new do |opts|
-  opts.banner = "'Usage: openstack_taster <test_suite> <image_name> <option>\n" \
-                "   or: openstack_taster <test_suite>\n" \
+  opts.banner = "Usage: openstack_taster <image_name> {suite_name} [--create-snapshot]\n" \
+                "   or: openstack_taster <image_name> [--create-snapshot]\n" \
                 '   or: openstack_taster <option>'
```

Comment:
This will work for `--help`, but if they attempt to use `--create-snapshot` it will throw an error.